src/components/accounts/recovery_setup/SetupRecoveryMethod.js

```diff
@@ -285,36 +300,49 @@ class SetupRecoveryMethod extends Component {
+        const { option, phoneNumber, email, success, emailInvalid, phoneInvalid, country, isNewAccount, settingUpNewAccount } = this.state;
         const { mainLoader, accountId, activeAccountId, ledgerKey, twoFactor } = this.props;
 
         if (!success) {
             return (
-                <StyledContainer className='small-centered'>
+                <StyledContainer className='small-centered border'>
                     <form onSubmit={e => {
                         this.handleNext();
                         e.preventDefault();
                     }}>
-                        <h1><Translate id='setupRecovery.header'/></h1>
-                        <h2><Translate id='setupRecovery.subHeader'/></h2>
-                        <h4><Translate id='setupRecovery.advancedSecurity'/></h4>
+                        <h1><Translate id='setupRecovery.header' /></h1>
+                        <h2><Translate id='setupRecovery.subHeader' /></h2>
+                        <h4>
+                            <Translate id='setupRecovery.advancedSecurity' />
+                            <Tooltip translate='profile.security.mostSecureDesc' icon='icon-lg'/>
+                        </h4>
                         <RecoveryOption
                             onClick={() => this.setState({ option: 'phrase' })}
                             option='phrase'
                             active={option}
                             disabled={this.checkDisabled('phrase')}
                         />
                         {(this.checkNewAccount() || !twoFactor) &&
-                        <RecoveryOption
-                            onClick={() => this.setState({ option: 'ledger' })}
-                            option='ledger'
-                            active={option}
-                            disabled={ledgerKey !== null && accountId === activeAccountId}
-                        />
+                            <RecoveryOption
+                                onClick={() => this.setState({ option: 'ledger' })}
+                                option='ledger'
+                                active={option}
+                                disabled={ledgerKey !== null && accountId === activeAccountId}
+                            />
                         }
-                        <h4><Translate id='setupRecovery.basicSecurity'/></h4>
+                        <h4>
+                            <Translate id='setupRecovery.basicSecurity' />
+                            <Tooltip translate='profile.security.lessSecureDesc' icon='icon-lg'/>
+                        </h4>
                         <RecoveryOption
-                            onClick={() => this.setState({ option: 'email' })}
+                            onClick={() => {
+                                this.setState({ option: 'email' });
+                                if (option !== 'email') {
```

Comment:
At first, I didn't understand why if the option is _not_ email, there would be an emailInput for us to focus? Is this using the correct magic string and ref?  But then I realised we're expecting `setState()` to have finished and the input to exist when this line runs, and we're actually checking `option` from the _pre-update state_.  For correctness, this focus code should exist in `componentDidUpdate()` or perhaps scheduled in the second, optional callback argument to the `setState()` call so that we're sure the update has been applied before we try to focus the input.  I'm not _entirely_ sure that even that would work, as I'm not positive that the DOM has been re-rendered or if only the state update has been completed when the callback fires.  It makes me wonder: could we just add an `autofocus` property to the input(s) that we care about focusing here and avoid this logic entirely?